packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -73,9 +73,25 @@ const EVENT_HANDLERS = [
     'keyup',
 ] as const;
 
+export const REGIONS = {
```

Comment:
This is kinda a poor-mans `enum` - could you make this a simple array of the values and drop the keys?

E.g.:
```
export const REGIONS = ['title', 'subtitle', ....] as const;

export type Region = (typeof REGIONS)[number];
```

Reply:
Yeah I've had this issue with repeated strings getting flagged by sonarqube. The solution I used was a proper enum like `enum Regions { Title = 'title', Subtitle = 'subtitle' }`.

The regions array type method helps constrain the type, but it leaves that sonarqube error. The enum is perhaps a little unnecessary in reality, but its better than having lots of `const title = 'title'` style things.

Reply:
Converted it to an enum